integration_tests/src/main/python/udf_cudf_test.py

```diff
@@ -12,15 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytest
 import pandas as pd
+import pytest
 import time
+from distutils.version import LooseVersion
 from typing import Iterator
 from pyspark.sql import Window
 from pyspark.sql.functions import pandas_udf, PandasUDFType
+from spark_init_internal import spark_version
 from spark_session import with_cpu_session, with_gpu_session
 from marks import allow_non_gpu, cudf_udf
 
+pytestmark = pytest.mark.skipif(LooseVersion(spark_version()) >= LooseVersion('3.1.0'),
+    reason="Pandas UDF on GPU tests don't support Spark 3.1.0+ yet")
+
```

Comment:
nit remove extra new line

Reply:
Actually two lines between two functions is recommended by the IntelliJ IDE.  :)